src/main/java/seedu/address/logic/commands/barchartresults/GenderBarChartCommandResult.java

```diff
@@ -0,0 +1,90 @@
+package seedu.address.logic.commands.barchartresults;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Map;
+import java.util.Objects;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.commands.CommandResult;
+
+/**
+ * Represents the command result for gender table generation.
+ */
+public class GenderBarChartCommandResult extends CommandResult {
+    private static final String FEEDBACKTOUSERMESSAGE = "A bar chart categorized by gender is shown";
```

Comment:
The final constant name might be clearer by separating words using underscores. 
Yet, would it be better to use the same name as other commands for success message, namely "MESSAGE_SUCCESS"?